### ISSUE TITLE
Issue Playing Streams

### ISSUE THREAD

**Issue by kpirnie** [sonroyaalmerol/m3u-stream-merger-proxy#95]
All streams fail to play stating the below.  All streams in my playlist do indeed play when not passed through the proxy/merger.

```
2024/08/23 06:55:39 Received request from 192.168.2.20:55433 for URL: /stream/ei1taWNrZXktYW5kLXRoZS1yb2Fkc3Rlci1yYWNlcnM=
2024/08/23 06:55:39 Channel not found from M3U_2: z - Mickey and the Roadster Racers
2024/08/23 06:55:39 [DEBUG] All streams skipped in lap 0
2024/08/23 06:55:39 Error reloading stream for z-mickey-and-the-roadster-racers: Error fetching stream. Exhausted all streams.
2024/08/23 06:55:39 Received request from 192.168.2.20:55434 for URL: /stream/ei1taWNrZXktYW5kLXRoZS1yb2Fkc3Rlci1yYWNlcnM=
2024/08/23 06:55:39 Channel not found from M3U_2: z - Mickey and the Roadster Racers
2024/08/23 06:55:39 [DEBUG] All streams skipped in lap 0
2024/08/23 06:55:39 Error reloading stream for z-mickey-and-the-roadster-racers: Error fetching stream. Exhausted all streams.
2024/08/23 06:55:39 Received request from 192.168.2.20:55435 for URL: /stream/ei1taWtlLWFuZC1tb2xseQ==.m3u8
```

**Comment by aniel300** [comment]
am getting the same error

**Comment by sonroyaalmerol** [comment]
The merged #96 should fix this issue. Let me know otherwise

**Comment by aniel300** [comment]
@sonroyaalmerol seems fixed for me, thanks.

**Comment by kpirnie** [comment]
> The merged #96 should fix this issue. Let me know otherwise

5 beers and a 1/8th cord of wood into a friday night mate ;)

will let ya know tomorrow 🤗🤗

**Comment by kpirnie** [comment]
Streams are back working :)
